modules/nf-core/mgikit/demultiplex/main.nf

```diff
@@ -0,0 +1,88 @@
+process MGIKIT_DEMULTIPLEX {
+    tag {"$run_id"}
+    label 'process_high'
+
+    conda "${moduleDir}/environment.yml"
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mgikit:0.1.6--h4c94732_0' :
+        'biocontainers/mgikit:0.1.6--h4c94732_0' }"
+
+    input:
+    tuple val(meta), path(samplesheet), path(run_dir)
+
+    output:
+    tuple val(meta), path("${prefix}/*.fastq.gz")                                                    , emit: fastq
```

Comment:
Some outputs are emitted twice as optional outputs and in `qc_reports` couldn't they just be emitted once?

Reply:
qc_reports contains multiple files and is mandatory while the other reports are optional and contain other files that are not needed for the qc report

Reply:
OK, then also fine